canister/src/api/get_block_headers.rs

```diff
@@ -0,0 +1,75 @@
+use ic_btc_interface::{GetBlockHeadersError, GetBlockHeadersRequest, GetBlockHeadersResponse};
+
+// Various profiling stats for tracking the performance of `get_block_headers`.
+#[derive(Default, Debug)]
+struct Stats {
```

Comment:
Is this struct being used at all? If not, I'd remove it and introduce it only whenever it becomes used.

Reply:
Done.